<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>MainWindow</class>
	<widget class="QMainWindow" name="MainWindow">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>532</width>
				<height>175</height>
			</rect>
		</property>
		<property name="windowTitle">
			<string>MainWindow</string>
		</property>
		<widget class="QWidget" name="centralwidget">
			<layout class="QGridLayout" name="gridLayout">
				<item row="1" column="3">
					<widget class="QComboBox" name="comboBox">
						<property name="sizeAdjustPolicy">
							<enum>QComboBox::AdjustToContents</enum>
						</property>
						<item>
							<property name="text">
								<string>+</string>
							</property>
						</item>
						<item>
							<property name="text">
								<string>-</string>
							</property>
						</item>
						<item>
							<property name="text">
								<string>*</string>
							</property>
						</item>
						<item>
							<property name="text">
								<string>/</string>
							</property>
						</item>
						<item>
							<property name="text">
								<string>//</string>
							</property>
						</item>
						<item>
							<property name="text">
								<string>%</string>
							</property>
						</item>
					</widget>
				</item>
				<item row="3" column="3">
					<widget class="QPushButton" name="pushButton">
						<property name="text">
							<string>Vypočítej</string>
						</property>
					</widget>
				</item>
				<item row="0" column="1">
					<widget class="QLabel" name="label">
						<property name="sizePolicy">
							<sizepolicy hsizetype="Preferred" vsizetype="Maximum">
								<horstretch>0</horstretch>
								<verstretch>0</verstretch>
							</sizepolicy>
						</property>
						<property name="text">
							<string>Vstup</string>
						</property>
					</widget>
				</item>
				<item row="2" column="4">
					<widget class="QLabel" name="label_3">
						<property name="sizePolicy">
							<sizepolicy hsizetype="Preferred" vsizetype="Maximum">
								<horstretch>0</horstretch>
								<verstretch>0</verstretch>
							</sizepolicy>
						</property>
						<property name="text">
							<string>[Příklad]</string>
						</property>
					</widget>
				</item>
				<item row="1" column="4">
					<widget class="QLineEdit" name="lineEdit_2"/>
				</item>
				<item row="1" column="1">
					<widget class="QLineEdit" name="lineEdit"/>
				</item>
				<item row="0" column="4">
					<widget class="QLabel" name="label_2">
						<property name="text">
							<string>Výsledek</string>
						</property>
					</widget>
				</item>
			</layout>
		</widget>
	</widget>
	<resources/>
	<connections>
		<connection>
			<sender>pushButton</sender>
			<signal>clicked()</signal>
			<receiver>MainWindow</receiver>
			<slot>say_hello()</slot>
			<hints>
				<hint type="sourcelabel">
					<x>269</x>
					<y>134</y>
				</hint>
				<hint type="destinationlabel">
					<x>168</x>
					<y>105</y>
				</hint>
			</hints>
		</connection>
	</connections>
	<slots>
		<slot>say_hello()</slot>
	</slots>
</ui>